Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
-FROM ubuntu:20.04
+FROM ubuntu:22.04
 
 RUN apt-get -y update && \
-    apt-get -y install pgbouncer=1.12.0-3 --no-install-recommends && \
+    apt-get -y install pgbouncer=1.16.1-1ubuntu1 --no-install-recommends && \
```

Comment:
JFMI, why are we pinning the exact package version here?
This will fail on the next (security) package upgrade on Ubuntu.

Reply:
The idea is to pin our charm releases to specific versions of the product, so in this case this container will be released as `pgbouncer:1.16-22.04`, as well as `pgbouncer:latest`. In the corresponding PR in the k8s repo, I'll be pinning the container to `pgbouncer:1.16-22.04`. Presumably this will also be updated to make use of the snap in future.